Mozilla.Firefox.ESR version 102.8.0

borque opened the pull request:
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/97184)

borque committed:
Mozilla.Firefox.ESR version 102.8.0

ItzLevvie reviewed (changes requested):
Make sure `InstallerLocale` corresponds to the ones shown in https://learn.microsoft.com/en-us/openspecs/office_standards/ms-oe376/6c085406-a698-4e12-9d4d-c3b0ee3dbc4a otherwise people won't be able to get the correct installer for their locale.

For example:
1) Japanese should be `ja-JP` instead of `ja-JA` (ja-JA doesn't exist and is not a valid locale)
2) Danish should be `da-DK` instead of `da-DA` (da-DA doesn't exist and is not a valid locale)

borque committed:
Update Mozilla.Firefox.ESR.installer.yaml

Corrected locals. I used some wrong when expanding from two letters to four letters encoding. Thanks @ItzLevvie !
